x-pack/plugins/uptime/server/lib/requests/get_latest_monitor.ts

```diff
@@ -53,11 +55,11 @@ export const getLatestMonitor: UMElasticsearchQueryFn<GetLatestMonitorParams, Pi
   };
 
   const result = await callES('search', params);
-  const ping: any = result.hits?.hits?.[0] ?? {};
-  const { '@timestamp': timestamp, ...monitorResult } = ping?._source ?? {};
+  const source = result.hits?.hits?.[0]?._source ?? {};
 
-  return {
-    timestamp,
-    ...monitorResult,
-  };
+  const decoded = PingType.decode({ ...source, timestamp: source['@timestamp'] });
+  if (isRight(decoded)) {
+    return decoded.right;
+  }
+  throw new Error(JSON.stringify(PathReporter.report(decoded)));
```

Comment:
i believe, it doesn't serve any purpose to do runtime validation both on client and server side. what do you think?